Makefile

```diff
@@ -11,6 +11,7 @@ clean: coverageclean typecoverageclean
 
 typecheck:
 	mypy --cobertura-xml-report typecover --html-report typecover .
+	mypy tests/integration/records
```

Comment:
I wanted to use types and check them in my integration tests when I found them useful, but not count them against my metrics.  This seems to do the trick!